src/api.v2/controllers/qcController.js

```diff
@@ -0,0 +1,64 @@
+// const { OK } = require('../../utils/responses');
+const AWSXRay = require('aws-xray-sdk');
+
+const { createQCPipeline } = require('../helpers/pipeline/pipelineConstruct');
+const handleQCResponse = require('../helpers/pipeline/handleQCResponse');
+
+const getLogger = require('../../utils/getLogger');
+const parseSNSMessage = require('../../utils/parse-sns-message');
+
+const logger = getLogger('[QCController] - ');
+
+const runQC = async (req, res) => {
```

Comment:
I was looking and comparing this `qcController` with `gem2sController`. I can see that in some respect, they are almost the same. For example, the handleResponse function in each of them is almost identical. That makes a lot of sense, what I don't understand is why there are differences between what `runQC` and what `runGem2s` return? Why for `runGem2s` it's fine to just have `res.json(OK())` and for `runQC` we have to return the `stateMachineArn` and `executionArn`? Shouldn't we be returning those also for runGem2s?

Reply:
You're right, I checked and runPipeline doesn't use the arns even for v1, so I'll make qc also return `OK()`